nixos/modules/services/x11/display-managers/lightdm.nix

```diff
@@ -72,17 +40,17 @@ let
       ${cfg.extraSeatDefaults}
     '';
 
-  gtkGreeterConf = writeText "lightdm-gtk-greeter.conf"
-    ''
-    [greeter]
-    theme-name = Adwaita
-    icon-theme-name = Adwaita
-    background = ${cfg.background}
-    '';
-
 in
 {
+  # Note: the order in which lightdm greeter modules are imported
+  # here determines the default: later modules (if enable) are
+  # preferred.
```

Comment:
I don't quite understand how this comment applies. If I also had a Qt greeter then I would have to explicitly disable the Gtk greeter in order to use the Qt one, right?